languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -66,6 +66,7 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
 	<!ENTITY couleur_invariable_m "noyer|canard"> <!-- fem: marine --> 
     <!ENTITY couleur_variable "alezan|ambré|aubère|bai|baillet|basané|beige|bigarré|bis|blafard|blond|blême|brillant|châtain|chiné|clair|cramoisi|dense|doré|écarlate|écru|fauve|fluorescent|foncé|glauque|incarnat|intense|livide|louvet|lumineux|mat|mauve|métallique|mordoré|opalin|orangé|pâle|perse|pourpre|profond|rosé|rose|rouan|rousse|roux|tourdille|tropical|vairon|vermeil|vernissé|vif|violet|zain|zinzolin">
     <!ENTITY couleur "blanc|bleu|brun|gris|jaune|noir|rouge|vert">
+    <!ENTITY adverbes_negation "pas|aucun(e(ment)?)?s?|jamais|guère|nulle(ment)?|plus|rien|personne">
```

Comment:
Would "point" fit in here, or would it do more harm than good?

Reply:
Indeed, I did not include 'point' in P_V_PAS because it generated a lot of FP due to confusion, but it should be relevant to include it in the entity, even to include an exception when using this entity